docs/TuxOwneeClass.md

```diff
@@ -0,0 +1,249 @@
+
+```javascript
+    owneeClassProps.anyPropTypes = {
+      message: React.PropTypes.object.isRequired,
+      deleteMessage: React.PropTypes.function.isRequired,
+      editMessage: React.PropTypes.function.isRequired
+    };
+```
+
+**NOTE** When performing prop validations with `anyPropTypes` that `props` in `this.props` are validated first, so they will override the validation of the `nearestOwnerProps`. For example, in the above scenario if `this.props.message` is an object, but `this.nearestOwnerProps.message` is a string, then the message property will be treated as valid since `this.props` takes priority.
+
+#### <a id="createOwneeClass-OwneeClass"></a>Return - `OwneeClass` - type: OBJECT [#](#createOwneeClass-OwneeClass)
+The `createOwneeClass` method returns a `TuxOwneeClass` which is an extension on top of a `React` Class. You can then simply use it to render components as you would any other `React` Class.
+
+##### <a id="OwneeClass-nearestOwnerProps"></a>Property - `OwneeClass.nearestOwnerProps` [#](#OwneeClass-nearestOwnerProps)
+Allows the `Ownee` to access the `nearestOwnerProps` on the closest `OwnerClass`.
+The `nearestOwnerProps` should be static, if you need to pass in dynamic `props` use the standard `React` `props` pass down syntax.
+
+**NOTE** `nearestOwnerProps` should be treated as immutable just like standard component props. Thus, you should never modify the `nearestOwnerProps` object. It is designed to store static methods and data.
+
+***
+
+##### <a id="TuxOwneeClass-Guidelines"></a>4) TuxOwneeClass Guidelines [#](#TuxOwneeClass-Guidelines)
+It is quite possible to never take advantage of the versatility that is provided by using the `nearestsOwnerProps`. Yet you would be missing out on the benefits that it provides. Passing properties down through a multitude of components is error prone, as it is easy to forget to pass all properties or methods that you need. Part of this difficulty can be alleviated by using `propTypes` to validate a components prop inputs and using the `React Spread` syntax to cascade down properties. However, even with those tools prop sharing is difficult and fragile. Additionally, it significantly hurts our ability to reuse components since our components now have to request `props` that they would not normally need just so that they can share those `props` with their children. Finally, cascading down `props` makes your `React` components vulnerable to refactoring/changes in component structure since you have to make sure you have the correct `props` in the correct places whenever you want to restructure your components. With `Tux`, as long as the 'nearestOwnerClasss` to the component doesn't change you can freely refactor your `React` structure without having to worry about losing access to your needed properties.
```

Comment:
Change components to component's in the fourth sentence.

In the last sentence correct spelling of `nearestOwnerClass` and change the ' to a \` in front of it.